### DISCUSSION TITLE
https://www.kelvinamoaba.live/blog/paying-attention-to-details

### DISCUSSION THREAD

**Discussion by giscus** [AmoabaKelvin/blog#3] [Announcements]
# https://www.kelvinamoaba.live/blog/paying-attention-to-details

 I recently got the chance to work with some individuals on a project. Most of the open source contributions I've done did not really have a specific set of designs to follow, so each contributor had 

https://www.kelvinamoaba.live/blog/paying-attention-to-details

**Comment by ABZABI** [comment]
Could you send a link to any design guide out there?

I’d like to have a look at a sample .

Is it like a pdf document or just the UI file in figma?🤔

**Reply by AmoabaKelvin** [reply]
It is currently closed source, so I can not share the designs with you; however, everything was done primarily in Figma.
On the first page, they went into great detail about the application's target audience and what it should do overall. Then came the components, which highlighted the spacing values and margins, among other things.

So the majority of the values that will be used in the actual screens were all listed on a few pages to make it easier to configure everything.
